Packs/FeedDHS/Integrations/DHSFeedV2/DHSFeedV2.yml

```diff
@@ -93,43 +93,21 @@ configuration:
   required: false
   type: 19
 - additionalinfo: 'The time interval for the first fetch (retroactive) in the following format: <number> <time
-    unit> of type minute/hour/day/year. For example, 1 minute, 12 hour.'
+    unit> of type minute/hour/day. For example, 1 minute, 12 hour. Limited to 48 hours.'
```

Comment:
If it is more than 1 (hour, minute, day) will it be hours/minutes/days? For example 3 minutes. 12 hours.